src/strategies/goldfinch-membership/index.ts

```diff
@@ -0,0 +1,74 @@
+import { multicall } from '../../utils';
+import _strategies from '..';
+
+export const author = 'blakewest';
+export const version = '0.1.0';
+
+const goListAbi = {
+  inputs: [
+    {
+      internalType: 'address',
+      name: '',
+      type: 'address'
+    }
+  ],
+  name: 'goList',
+  outputs: [
+    {
+      internalType: 'bool',
+      name: '',
+      type: 'bool'
+    }
+  ],
+  stateMutability: 'view',
+  type: 'function'
+};
+
+const LEGACY_GOLDFINCH_CONFIG = '0x4eb844Ff521B4A964011ac8ecd42d500725C95CC';
+const UID = '0xba0439088dc1e75F58e0A7C107627942C15cbb41';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const strategyFn = _strategies['erc1155-all-balances-of'].strategy;
```

Comment:
Would be great if you can change this to import just `erc1155-all-balances-of` strategy into this
```
import strategy as erc1155AllBalancesOf from '../erc1155-all-balances-of';
```